marlowe-playground-client/src/MainFrame/State.purs

```diff
@@ -274,8 +278,33 @@ handleAction s (HaskellAction action) = do
       selectView BlocklyEditor
     _ -> pure unit
 
+handleAction s (JavascriptAction action) = do
+  toJavascriptEditor (JavascriptEditor.handleAction s action)
+  case action of
+    JS.SendResultToSimulator -> do
+      mContract <-
+        peruse
+          ( _javascriptState
+              <<< JS._compilationResult
+              <<< JS._CompiledSuccessfully
+              <<< JSI._result
+          )
+      let
+        contract = case mContract of
+          Just con -> show $ pretty con
+          _ -> ""
```

Comment:
```purescript
maybe "" (show <<< pretty) mContract
```

...perhaps?

Reply:
I never used `maybe` before but since seeing @palas use it a few times I'm warming to it, this looks like a nice use case

Reply:
moved this into a `Getter` so it doesn't quite make sense now